swig/sundials/fsundials_errors_mod.i

```diff
@@ -0,0 +1,30 @@
+// ---------------------------------------------------------------
+// Programmer: Cody J. Balos @ LLNL
+// ---------------------------------------------------------------
+// SUNDIALS Copyright Start
+// Copyright (c) 2002-2022, Lawrence Livermore National Security
+// and Southern Methodist University.
+// All rights reserved.
+//
+// See the top-level LICENSE and NOTICE files for details.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+// SUNDIALS Copyright End
+// ---------------------------------------------------------------
+// Swig interface file
+// ---------------------------------------------------------------
+
+%module fsundials_errors_mod
+
+%include "../sundials/fsundials.i"
```

Comment:
It looks like this line is leftover from before, and for consistency with the other files should be:
```suggestion
// Load the typedefs and generate a "use fsundials_types_mod" statement in the module
%import "../sundials/fsundials_context_mod.i"
%import "../sundials/fsundials_types_mod.i"

%include "../sundials/fcopyright.i"
```